docs/docsite/rst/community/how_can_I_help.rst

```diff
@@ -1,33 +1,42 @@
 How To Help
 ===========
 
-There are many ways to help the Ansible project. 
+.. contents:: Topics
+
+There are many ways to help the Ansible project.
 
 Become a power user
 -------------------
 
-A great way to help the Ansible project is to become a power user. Use Ansible everywhere you can. Take tutorials and classes. Read the `official documentation <http://docs.ansible.com/ansible/latest/index.html>` and some of the `many excellent books <https://www.amazon.com/s/ref=nb_sb_ss_c_2_7?url=search-alias%3Dstripbooks&field-keywords=ansible&sprefix=ansible%2Caps%2C260>` about Ansible. `Get certified <https://www.ansible.com/training-certification>`.
+A great way to help the Ansible project is to become a power user:
+
+* Use Ansible everywhere you can
+* Take tutorials and classes
+* Read the `official documentation <http://docs.ansible.com/ansible/latest/index.html>`_
+* Study some of the `many excellent books <https://www.amazon.com/s/ref=nb_sb_ss_c_2_7?url=search-alias%3Dstripbooks&field-keywords=ansible&sprefix=ansible%2Caps%2C260>`_ about Ansible
+* `Get certified <https://www.ansible.com/training-certification>`_.
 
 When you become a power user, your ability and opportunities to help the Ansible project in other ways will multiply quickly.
 
 Ask and answer questions online
 -------------------------------
 
-There are many forums online where Ansible users ask and answer questions. Reach out and communicate with your fellow Ansible users. Ask good questions, and give good answers. 
+There are many forums online where Ansible users ask and answer questions. Reach out and communicate with your fellow Ansible users. Ask good questions, and give good answers.
```

Comment:
Are there *bad* questions? Please consider rephrasing.

Reply:
There are bad questions, actually. We could consider linking to this: http://www.catb.org/esr/faqs/smart-questions.html

But for now, let's just delete that "Ask" sentence.